test/test_calendar.py

```diff
@@ -150,3 +156,107 @@ def test_parse_date_offset_no_offset_magnitude():
 
     expected_error = "No numerical value given for offset: YS"
     assert str(exc_info.value) == expected_error
+
+
+@pytest.mark.parametrize(
+        "start_date, end_date, caltype_int, expected",
+        [
+            (
+                datetime.datetime(year=4, month=1, day=1),
+                datetime.datetime(year=5, month=1, day=1),
+                GREGORIAN,
+                366 * SEC_PER_DAY
+            ),
+            (
+                datetime.datetime(year=4, month=1, day=1),
+                datetime.datetime(year=5, month=1, day=1),
+                NOLEAP,
+                365 * SEC_PER_DAY
+            ),
+            (
+                datetime.datetime(year=300, month=1, day=1),
+                datetime.datetime(year=301, month=1, day=1),
+                GREGORIAN,
+                365 * SEC_PER_DAY
+            ),
+            (
+                datetime.datetime(year=400, month=1, day=1),
+                datetime.datetime(year=401, month=1, day=1),
+                GREGORIAN,
+                366 * SEC_PER_DAY
```

Comment:
```suggestion
                datetime.datetime(year=400, month=1, day=1),
                datetime.datetime(year=401, month=12, day=31),
                GREGORIAN,
                365 * SEC_PER_DAY
```
This tests that month and day increments are working also

Reply:
Thanks great idea - added this one in!